### PR TITLE
Style fix: align pointer dereference operator to right

### PR DESCRIPTION
**Pull request by LaszloLango**
JerryScript-DCO-1.0-Signed-off-by: László Langó llango.u-szeged@partner.samsung.com

### CODE REVIEW

File: tools/vera++/scripts/rules/jerry_dereference_operator_awlays_on_right.tcl [hunk 1]
```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/tclsh
```
Comment (zherczeg) [inline tools/vera++/scripts/rules/jerry_dereference_operator_awlays_on_right.tcl]: jerry_dereference_operator_awlays_on_right.tcl

awlays?

Comment (LaszloLango) [inline tools/vera++/scripts/rules/jerry_dereference_operator_awlays_on_right.tcl]: As far as I know this is the correct style that should be followed. Do you know any exceptions?

Comment (zherczeg) [inline tools/vera++/scripts/rules/jerry_dereference_operator_awlays_on_right.tcl]: The file name is bad :)

Comment (LaszloLango) [inline tools/vera++/scripts/rules/jerry_dereference_operator_awlays_on_right.tcl]: is missed that :)

**Comment by zherczeg** [comment]
LGTM. Can we extend this to type casting in a followup patch? E.g. (type *) ?

**Comment by LaszloLango** [comment]
@zherczeg, sure

**Comment by ruben-ayrapetyan** [comment]
Looks good to me

**Pull request merged**